pino.js

```diff
@@ -338,6 +344,18 @@ function pino (opts, stream) {
     }
   }
 
+  var base = (typeof iopts.base === 'object') ? iopts.base : defaultBase
+
+  if (iopts.name !== undefined) {
+    base = Object.assign({}, base, {
+      name: iopts.name
+    })
+  }
+
+  if (base != null) {
+    instance = instance.child(base)
```

Comment:
The default instance should not be a child.

cc: @davidmarkclements @mcollina

Reply:
I was following the comments from https://github.com/pinojs/pino/pull/282#pullrequestreview-57188442. Unfortunately I don't know enough about Pino to understand the implications of using a child logger for the default instance. When I get a moment I'll dig deeper into Pino.

Reply:
It seems odd to me. But that's why I pinged the other two, so they can provide enlightenment.

Reply:
why not? I think it's the best way to do it. Considering that pino children do not keep a reference to their parent, I see no obvious problem with this.

Reply:
The only side effect of this is that it makes the base-log properties overridable by a log line. I don't think it's a bad thing: https://github.com/pinojs/pino/blob/master/pino.js#L136-L138.

Reply:
Okay.